gctscript/examples/ta/rsi.gct

```diff
@@ -0,0 +1,13 @@
+fmt := import("fmt")
+exch := import("exchange")
+t := import("times")
+ind := import("indicator/rsi")
+
+load := func() {
+   start := t.add(t.now(), -t.hour*24*30)
+   ohlcvData := exch.ohlcv("BTC Markets", "BTC-AUD", "-", "", start, t.now(), t.hour*24)
```

Comment:
This returns "invalid interval". Changing it to "1d" was fine and returned:
`[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 28.789019503744907, 27.85788150182829, 35.80734224157345, 37.28104027826195, 33.85346922663619, 25.53771880851688, 24.96625290971767, 27.322125454564418, 28.73015163227809, 14.273864967084151, 24.9228886313506, 22.883391936295734, 25.810
349095174974, 24.39493906593634, 30.842185479488926, 33.649405309657524, 46.40264190029889]
 Shutting down script: rsi.gct ID: 8eb188c7-396f-4714-8b45-3abaa13fd8d8`

Reply:
ah yes i forgot to update some of the other scripts interval examples will do that now! thanks